Extensions/dot/mcclim-dot.asd

```diff
@@ -0,0 +1,38 @@
+(in-package #:asdf-user)
+
+(defsystem "mcclim-dot"
+  :license "MIT"
+  :description "Support for using DOT based graph layout engines."
+  :depends-on ("mcclim-dot/core" "mcclim-dot/graphviz")
+  :in-order-to ((test-op (test-op "mcclim-dot/test"))))
+
+(defsystem "mcclim-dot/core"
+  :license "MIT"
+  :description "Core DOT routines."
+  :depends-on ("mcclim"
+               "mcclim-bezier"
```

Comment:
you may want to use a more complete alternative that has been merged recently into core: draw-bezigon* et al.

Reply:
Nice. Done.